packages/core/src/Selection.js

```diff
@@ -208,6 +209,23 @@ export class Selection extends Param {
     return super.update(this._resolved);
   }
 
+  /**
+   * Reset the selection state by removing all provided clauses. If no clause
+   * array is provided as an argument, all current clauses are removed. The
+   * reset method (if defined) is invoked on all corresponding clause sources.
+   * The reset is relayed to downstream selections that include this selection.
+   * @param {SelectionClause[]} [clauses] The clauses to remove. If
+   *  unspecified, all current clauses are used as the default.
```

Comment:
```suggestion
   *  unspecified, all current clauses are removed.
```

Feels slightly clearer